Jenkinsfile

```diff
@@ -1,26 +1,45 @@
 pipeline {
     agent { label 'mobile-builder-ios-pull-request'  }
-    environment {
-       PRODUCT = 'couchbase-lite-ios-ee'
-   }
     stages {
         stage('Checkout'){
             steps {
                 sh """
-                    git clone https://github.com/couchbaselabs/${env.PRODUCT}.git
-                    pushd ${env.PRODUCT}
+                    #!/bin/bash
+                    set -e
+                    shopt -s extglob dotglob
+
+		    # move PR related repo to tmp folder
+		    mkdir tmp
+                    mv !(tmp) tmp
+
+		    # clone and update submodules here
+                    git clone https://github.com/couchbaselabs/couchbase-lite-ios-ee.git --branch $CHANGE_TARGET
+		    # submodule update inside lite-ios
+		    pushd couchbase-lite-ios-ee
                     git submodule update --init --recursive
-                    ./Scripts/prepare_project.sh
-                    cd couchbase-lite-ios
-		    git checkout master
-		    git pull origin master
-                    popd
+		    popd
+
+		    # restructure folders
+		    mv -v couchbase-lite-ios-ee/* .
+		    rsync -a tmp/ couchbase-lite-ios/
```

Comment:
I'm not sure if --delete is needed to completely replace all content inside couchbase-lite-ios. 

Alternatively but two commands,
```
rm -rf couchbase-lite-ios && mv tmp couchbase-lite-ios
```

Reply:
i tried `-f ` flag, but not worked. it should force replace all contents. I will try with the two commands, which will be easy to understand.